components/viaduct/android/src/main/java/mozilla/appservices/httpconfig/HttpConfig.kt

```diff
@@ -54,6 +54,17 @@ object RustHttpConfig {
         }
     }
 
+    /** Allows connections to the hard-coded address the Android Emulator uses for
+      * localhost (ie, http://10.0.2.2) - if you don't call this, viaduct will fail to
```

Comment:
```suggestion
      * the host's localhost (ie, http://10.0.2.2) - if you don't call this, viaduct will fail to
```

or something like that? Or leave out `localhost` and call it `the host machine`?